### ISSUE TITLE
Browser-side build of Comunica 2.4.0 overrides global process variable even if already set

### ISSUE THREAD

**Issue by jacoscaz** [comunica/comunica#1050]
#### Issue type:

- :bug: Bug

____
#### Description:

Comunica 2.4.0 is breaking `quadstore-comunica` when used server-side due to the override of the global `process` variable at: https://github.com/comunica/comunica/blob/b84898f73a1a2c1fc413b51b2facacfd6bb729e4/packages/actor-init-query/lib/ActorInitQuery-browser.ts#L1-L8

`quadstore-comunica` is intentionally built with Webpack 5.x targeting the `web` platform as that produces an isomorphic build that works well in both browser and server-side environments. However, the override is now preventing this build from being usable within Node.js.

Simple fix: `global.process = typeof process !== 'undefined' ? process : require('process');` . Should I open a PR?

____
#### Environment:

```
| software         | version
| ---------------- | -------
| Comunica Engine  | 2.4.0 
| node             | v16.13.0
| npm              | 8.1.0
| yarn             | 1.22.17
| Operating System | darwin (Darwin 21.4.0)
```

____
#### Crash log:

N/A

**Comment by rubensworks** [comment]
Indeed, this was one aspect I was not very happy about, but there's not really a good workaround available.

But yet, feel free to open a PR with your proposed fix!

**Comment by rubensworks** [comment]
> quadstore-comunica is intentionally built with Webpack 5.x targeting the web platform as that produces an isomorphic build that works well in both browser and server-side environments.

And with the new changes, I guess this workaround won't be needed anymore?

**Comment by jacoscaz** [comment]
> And with the new changes, I guess this workaround won't be needed anymore?

Well, I guess it depends on Comunica's own plans. For me, it's a lot easier to maintain a single build targeting the subset of features that is common to all runtimes (browser-side and server-side) and therefore I'd rather keep `quadstore-comunica` as-is. But, I do understand that this is an opinionated view. I'll open a PR with the fix suggested above and if that doesn't cut it I'll try something different on my end.

**Comment by rubensworks** [comment]
Fixed in #1051
